Paper: Accelerating the Advancement of Data Science Education (#478)
scipy-conference/scipy_proceedings: papers/Amal_Bhatnagar/AcceleratingtheAdvancementofDataScienceEducation.rst

as the “Workshop on Theoretical Foundations of Data Science,” “The Park
City Math Institute 2016 Summer Undergraduate Faculty Program,” and
“Envisioning the Data Science Discipline: The Undergraduate
Perspective.” While these workshops produced comprehensive guidelines on
the structure of the programs, the actual content and teaching
modalities remain unclear.

On June 24-27, 2019, the Division of Data Sciences at the UC Berkeley
hosted the 2nd National Workshop on Data Science Education which brought
together nearly 70 faculty from a diverse range of higher-education
institutions on at different stages of data science education. As a
pioneer in undergraduate data science education, UC Berkeley shared its
comprehensive set of open-license and open-source resources that range
from teaching materials to cloud infrastructure at the workshop. 

The workshop sought to build a national community of practice around
undergraduate Data Science education by focusing around four primary
areas:

*1)* Examining the *Foundations of Data Science (Data 8)* course - How
does the content to pedagogical methods of Data 8 integrates various
disciplines through the use of computational and inferential

> thinking.*

Reviewer: ```suggestion
thinking.
```

This `*` was unbalanced

*2)* Showcase the infrastructural platform that Berkeley has developed
for its courses, and empower participants to use its many open-source
components to overcome the *financial and technical barriers.*

*3)* Applying Modular Data Science education content into various
disciplinary fields past the scope of that from a traditional computer

> science or statistics courses provide.*

Reviewer: ```suggestion
science or statistics courses provide.
```

All of this infrastructure is crucial for creating, deploying, and
grading data science homework and lab assignments. Having this
educational cyber-infrastructure is more efficient than local
infrastructure, as instructors can teach students for many, the system
holds all the necessary material, simplifies data management and
analysis, and visualizes data for instructors. Before Berkeley launched
its integrated system, the teaching faculty found it difficult to
efficiently scale courses at the rate of their increasing interest.
Berkeley’s adoption of JupyterHub has allowed more than 1,600 students
to enroll in Data 8 for its Spring 2019 iteration, a historic milestone
that would not have been possible absent Berkeley’s educational
cyber-infrastructure. 

Creating and Incorporating Modular Data Science Content 
--------------------------------------------------------

There are two main concerns when modularizing data science content:
*Having just one introductory data science class is not enough to
warrant an entire data science curricula, and creating a sustainable
model that supports the data science curricula is challenging for newly

> adopting institutions.  *

Reviewer: ```suggestion
adopting institutions.*
```

Human Contexts and Ethics of Data,* which covers the implications of
computing, such as algorithmic bias. Additionally, students can take
*Computer Science 294: Fairness in Machine Learning,* which spends a
semester in resisting racial, political, and physical discrimination.
Faculty have also come together to create the Algorithmic Fairness and
Opacity Working Group at Berkeley’s School of Information that
brainstorms methods to improve algorithms’ fairness, interpretability,
and accountability. Implementing such courses and interdisciplinary
groups is key to start the conversation within academic institutions, so
students can mitigate such algorithmic bias when they work in industry
or academia post-graduation. 

Databases and algorithms are socio-technical objects; they emerge and
evolve in tandem with the societies in which they operate [Latour 90]_.
Understanding data science in this way and recognizing its social
implications requires a different kind of critical thinking that is
taught in data science courses. Issues such as computational agency
[Tufekci 15]_, the politics of data classification and statistical

> inference [Bowker 08]_, [Desrosières 11]_, and the perpetuation

Reviewer: ```suggestion
inference [Bowker 08]_, [Desrosieres 11]_, and the perpetuation
```

matching reference to below changes

add ethical standards to their mission statements or to hire chief
ethics officers to oversee not only day-to-day operations but also the
larger social consequences of their work. 

Summary & Vision
----------------

We envision a world where all students can learn ethical data-driven
techniques regardless of their domain and can manipulate data to find
better solutions to problems. To do that requires a four part strategy
involving creating a campus wide foundational data science course, the
modularization of data science course content to integrate it with
courses in existing domains, the scalable cloud infrastructure power it
all, and the human context and ethics content to reign in misuse of data
& artificial intelligence. Integrating Python across different fields
exposes students to learning programming in areas they would not have
previously expected. These strategies will accelerate the creation of a
space for Data Science to exist as a cross-campus endeavor and engage
faculty and students in different departments.

References
----------

.. [Baumer 17] Baumer, B. S., Kaplan, D. T., & Horton, N. J. (2017). Modern Data

> Science with R. Retrieved from http://mdsr-book.github.io/	

Reviewer: ```suggestion
   Science with R. Retrieved from http://mdsr-book.github.io/	
```
All of these lines need to be indented or RST gets upset

.. [Bowker 08] Bowker, G. C., & Star, S. L. (2008). Sorting things out: Classification

> and its consequences. Cambridge, MA: MIT Press.	

Reviewer: ```suggestion
   and its consequences. Cambridge, MA: MIT Press.	
```

> .. [Desrosières 11] Desrosières, A. (2011). The politics of large numbers: A history of

Reviewer: ```suggestion
.. [Desrosieres 11] Desrosieres, A. (2011). The politics of large numbers: A history of
```

This is terrible, but `pdflatex` has trouble with unicode and the most expedient way to handle it is to remove the composed character

Reviewer 2: I'm not so sure about this one. We should be able to build papers with è in them:

<img width="311" alt="Screenshot 2019-07-08 06 59 24" src="https://user-images.githubusercontent.com/6868396/60808513-1c8c7980-a14e-11e9-99a9-53bbe00ddaf7.png">

> statistical reasoning. Cambridge, MA: Harvard University Press.	

Reviewer: ```suggestion
   statistical reasoning. Cambridge, MA: Harvard University Press.	
```

.. [Eubanks 19] Eubanks, V. (2019). AUTOMATING INEQUALITY: How high-tech tools profile,

> police, and punish the poor. PICADOR.	

Reviewer: ```suggestion
   police, and punish the poor. PICADOR.	
```

.. [Latour 90] Latour, B. (1990). Technology is society made durable. The Sociological

> Review, 38(1), supplement, 103-131. Retrieved from

Reviewer: ```suggestion
   Review, 38(1), supplement, 103-131. Retrieved from
```

> http://www.bruno-latour.fr/sites/default/files/46-TECHNOLOGY-DURABLE-GBpdf.pdf	

Reviewer: ```suggestion
   http://www.bruno-latour.fr/sites/default/files/46-TECHNOLOGY-DURABLE-GBpdf.pdf	
```

.. [NASEMS 18] National Academies of Sciences, Engineering, and Medicinemies of

> Sciences. (2018, May 02). Data Science for Undergraduates: Opportunities

Reviewer: ```suggestion
   Sciences. (2018, May 02). Data Science for Undergraduates: Opportunities
```

> and Options. Retrieved from https://doi.org/10.17226/25104	

Reviewer: ```suggestion
   and Options. Retrieved from https://doi.org/10.17226/25104	
```

.. [Noble 18] Noble, S. U. (2018). Algorithms of oppression how search engines

> reinforce racism. New York: New York University Press.	

Reviewer: ```suggestion
   reinforce racism. New York: New York University Press.	
```

.. [ONeil 18] ONeil, C. (2018). Weapons of math destruction: How big data increases

> inequality and threatens democracy. London: Penguin Books.	

Reviewer: ```suggestion
   inequality and threatens democracy. London: Penguin Books.	
```

.. [Tufekci 15] Tufekci, Z. (2015). Algorithmic harms beyond Facebook and Google:

> Emergent challenges of computational agency. Colorado Technology Law

Reviewer: ```suggestion
   Emergent challenges of computational agency. Colorado Technology Law
```

> Journal, 203-218. Retrieved from

Reviewer: ```suggestion
   Journal, 203-218. Retrieved from
```

> https://ctlj.colorado.edu/wp-content/uploads/2015/08/Tufekci-final.pdf.	

Reviewer: ```suggestion
   https://ctlj.colorado.edu/wp-content/uploads/2015/08/Tufekci-final.pdf.	
```

outlines and curriculum. Institutional IT staff members would have to
obtain additional training, which would vary across institutions to
better fit the differing needs and implementations of the data science
courses causing inconsistency. For many small institutions and
universities, this proves to be a major barrier in course delivery. The
development of regional or national cloud-based computing solutions that
can serve individual educational institutions is needed.

Universities must invest resources into developing data science
educational infrastructure like JupyterHub, a platform not many
universities have and differs from research cyber-infrastructure. The
two have different goals, resource needs, deployment timelines, cost and
pricing of models, and broad access mandate. Data science educational
infrastructure is deployed for relatively low resource use by a large
number of relatively unsophisticated users. Making the data science
infrastructure accessible requires establishing three components. At UC
Berkeley, the core components include setting up a campus wide
JupyterHub, integration with existing campus Learning Management Systems
(LMS), e.g. Canvas

> (`*https://www.instructure.com/* `https://www.instructure.com/>`__), and

Reviewer: ```suggestion
(`*https://www.instructure.com/*_ `<https://www.instructure.com/>`__), and
```

My RST is a bit rusty, but this should at least clear the error.

You can run `rst-lint` locally to find other issues if you first either

`pip install restructured-text`
or
`conda install -c conda-forge restructured-text`

together nearly 70 faculty from a diverse range of higher-education
institutions on at different stages of data science education. As a
pioneer in undergraduate data science education, UC Berkeley shared its
comprehensive set of open-license and open-source resources that range
from teaching materials to cloud infrastructure at the workshop. 

The workshop sought to build a national community of practice around
undergraduate Data Science education by focusing around four primary
areas:

*1)* Examining the *Foundations of Data Science (Data 8)* course - How
does the content to pedagogical methods of Data 8 integrates various
disciplines through the use of computational and inferential
thinking.

*2)* Showcase the infrastructural platform that Berkeley has developed
for its courses, and empower participants to use its many open-source
components to overcome the *financial and technical barriers.*

*3)* Applying Modular Data Science education content into various
disciplinary fields past the scope of that from a traditional computer
science or statistics courses provide.

*4)* Implementing Ethical Content into *data science courses and

> examines how such integration could work.*

Reviewer 3: ```suggestion
examining how such integration could work.*
```

computing, such as algorithmic bias. Additionally, students can take
*Computer Science 294: Fairness in Machine Learning,* which spends a
semester in resisting racial, political, and physical discrimination.
Faculty have also come together to create the Algorithmic Fairness and
Opacity Working Group at Berkeley’s School of Information that
brainstorms methods to improve algorithms’ fairness, interpretability,
and accountability. Implementing such courses and interdisciplinary
groups is key to start the conversation within academic institutions, so
students can mitigate such algorithmic bias when they work in industry
or academia post-graduation. 

Databases and algorithms are socio-technical objects; they emerge and
evolve in tandem with the societies in which they operate [Latour90]_.
Understanding data science in this way and recognizing its social
implications requires a different kind of critical thinking that is
taught in data science courses. Issues such as computational agency
[Tufekci15]_, the politics of data classification and statistical
inference [Bowker08]_, [Desrosieres11]_, and the perpetuation
of social injustice through algorithmic decision making [Eubanks19]_, 

> [Noble 18]_, [ONeil18]_ are well known to scholars in the

Reviewer 3: ```suggestion
[Noble18]_, [ONeil18]_ are well known to scholars in the
```
